synapse/replication/slave/storage/room.py

```diff
@@ -12,26 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from synapse.replication.tcp.streams import PublicRoomsStream
 from synapse.storage.database import DatabasePool
 from synapse.storage.databases.main.room import RoomWorkerStore
 
 from ._base import BaseSlavedStore
-from ._slaved_id_tracker import SlavedIdTracker
 
 
 class RoomStore(RoomWorkerStore, BaseSlavedStore):
     def __init__(self, database: DatabasePool, db_conn, hs):
         super().__init__(database, db_conn, hs)
```

Comment:
The `__init__` can be fully removed if it just calls `super(...)`. 🚗 

Same seems to be true of `process_replication_rows`.